test-e2e/config-import.js

```diff
@@ -64,6 +64,65 @@ test('config import - load and re-load config manually', async (t) => {
   )
 })
 
+test('deletion of data before loading a new config', async (t) => {
+  const manager = createManager('device0', t)
+  const project = await manager.getProject(await manager.createProject())
+  let nPresets = (await project.preset.getMany()).length
+  for (let i = 0; i < 10; i++) {
+    await project.importConfig({
+      configPath: defaultConfigPath,
+    })
+    nPresets = (await project.preset.getMany()).length
+    await project.importConfig({
+      configPath: './tests/fixtures/config/validConfig.zip',
+    })
+    await project.importConfig({
+      configPath: defaultConfigPath,
+    })
+    assert.equal(
```

Comment:
This is a failing test that I'm not being able to solve. I basically:
* load default config and take note of the number of presets
* load another config
* load the default config again
* Expect the number of presets to be the same as whe the first default config was loaded
The actual number of presets is `nPresetsDefaultConfig + nPresetsValidConfig`. 
If I run it in a loop, I get the same number every time (so presets are being deleted...)

Reply:
This was just me misunderstanding Promises. Before I did:
`Promise.all([[presetsToDelete], [fieldsToDelete], [translationsToDelete]].flat())`
(basically flatting the list of stuff to delete and passing it to `Promise.all`)
Now I do:
`Promise.all([Promise.all(presetsToDelete), Promise.all(fieldsToDelete), Promise.all(translationsToDelete)])`

I think I'm not doing deletions of `presets`, `fields`, `translations` in parallel now (only within themselves, so `preset` deletion is in parallel, `field` deletion too, etc)

@EvanHahn any thought?